templates/errorpage/errorpage.css

```diff
@@ -0,0 +1,7 @@
+.errorpage h1 {
+    font-size: 48px;
```

Comment:
@helms-charity do you think I can make this global ?

Reply:
no, if anything it would be 36px
![image](https://github.com/user-attachments/assets/baf005c1-9c2f-4c26-bb1d-64d28681ad19)

Reply:
done